app/Http/Controllers/Account/AccountController.php

```diff
@@ -53,6 +53,18 @@ public function deleteUpload(Request $request, Upload $upload)
         return redirect()->route('account.uploads');
     }
 
+    public function deleteAllUploads(Request $request)
+    {
+        $uploads = $request->user()->uploads()->get();
```

Comment:
I think this suffers the N+1 problem. We don't actually need the user, so we should just query for uploads with the current user's ID directly rather than going through the user's relation.